Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/Encrypter.java

```diff
@@ -7,35 +7,57 @@
  */
 package com.breadwallet.corecrypto;
 
+import com.breadwallet.corenative.crypto.BRCryptoCipher;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
 /* package */
 final class Encrypter implements com.breadwallet.crypto.Encrypter {
 
-    // TODO(fix): Implement me!
-
     /* package */
     static Encrypter createForAesEcb(byte[] key) {
-        return null;
+        BRCryptoCipher cipher = BRCryptoCipher.createAesEcb(key).orNull();
```

Comment:
I think we'd plan on renaming this class.  Worth doing now?

Reply:
Certainly. Do you want the Swift updated as well? Will cause a (easily resolvable) break in the iOS app.